test/commands/pexpire.js

```diff
@@ -17,7 +18,10 @@ describe('pexpire', () => {
 
         return redis.pttl('foo');
       })
-      .then(result => expect(result).toBeGreaterThanOrEqualTo(1));
+      .then(result => expect(result).toBeGreaterThanOrEqualTo(1))
+      .then(() => Promise.delay(200))
+      .then(() => redis.get('foo'))
+      .then(result => expect(result).toBe(null));
```

Comment:
Thanks for taking the time to add these integration tests (the more we have of those the better!) 👍 🎉